src/cs/building/PowerPlant.cs

```diff
@@ -332,7 +332,7 @@ public partial class PowerPlant : Node2D {
 
 		// Check if the multiplier window should be shown
 		if(MultiplierMax <= 1) {
```

Comment:
if the `if` branch becomes useless, just replace the `else` branch with if of the negated condition, i.e.
```cs
if(MultiplierMax > 1) { /* else branch body */ }
```